config/activityLog.js

```diff
@@ -48,5 +48,14 @@ module.exports = {
       user: 'None',
       workTime: 'None',
     },
+    queueStats: {
+      minute: 'None',
```

Comment:
The log line gets timestamped when written out so that field will always be the first one, so let's preserve alphabetical order of our fields, which all get written as the log record's "message" anyway